main.tf

```diff
@@ -105,52 +103,109 @@ module "client" {
   client_tags              = var.client_tags
   client_access_config     = var.client_access_config
   client_private           = var.client_private
+  client_iam_module        = module.client_iam
+  client_gcs_module        = module.client_gcs
+  client_config_module     = module.client_config
 
   services = google_project_service.main.*.service
 }
 
 module "server" {
   source = "./modules/server"
 
-  enable_cai_bucket                                   = var.enable_cai_bucket
-  project_id                                          = var.project_id
-  gsuite_admin_email                                  = var.gsuite_admin_email
-  forseti_version                                     = var.forseti_version
-  forseti_repo_url                                    = var.forseti_repo_url
+  project_id               = var.project_id
+  forseti_version          = var.forseti_version
+  forseti_repo_url         = var.forseti_repo_url
+  forseti_home             = var.forseti_home
+  forseti_run_frequency    = var.forseti_run_frequency
+  server_type              = var.server_type
+  server_region            = var.server_region
+  server_boot_image        = var.server_boot_image
+  server_boot_disk_size    = var.server_boot_disk_size
+  server_boot_disk_type    = var.server_boot_disk_type
+  server_tags              = var.server_tags
+  server_access_config     = var.server_access_config
+  server_private           = var.server_private
+  cloudsql_proxy_arch      = var.cloudsql_proxy_arch
+  network                  = var.network
+  subnetwork               = var.subnetwork
+  network_project          = var.network_project
+  server_grpc_allow_ranges = var.server_grpc_allow_ranges
+  server_ssh_allow_ranges  = var.server_ssh_allow_ranges
+  server_instance_metadata = var.server_instance_metadata
+  suffix                   = local.random_hash
+
+  policy_library_home                    = var.policy_library_home
+  policy_library_repository_url          = var.policy_library_repository_url
+  policy_library_sync_enabled            = var.policy_library_sync_enabled
+  policy_library_sync_gcs_directory_name = var.policy_library_sync_gcs_directory_name
+  policy_library_sync_git_sync_tag       = var.policy_library_sync_git_sync_tag
+  policy_library_sync_ssh_known_hosts    = var.policy_library_sync_ssh_known_hosts
+
+  client_iam_module    = module.client_iam
+  cloudsql_module      = module.cloudsql
+  server_config_module = module.server_config
+  server_gcs_module    = module.server_gcs
+  server_iam_module    = module.server_iam
+  server_rules_module  = module.server_rules
+
+  services = google_project_service.main.*.service
+}
+
+module "cloudsql" {
```

Comment:
Does this module also need to include the variable `network`? 

https://github.com/forseti-security/terraform-google-forseti/blob/1d2a79c3d43b6b285863dcc7595e8f0a1ec14711/modules/cloudsql/variables.tf#L20

Reply:
yes